vision/roadmap.md

```diff
@@ -21,6 +21,18 @@ Users will be able to send messages to group chats with 2+ participants.
 
 Teams will be able to read and send encrypted DMs and announcements using a shared team identity. The account admin will be able to delegate access to and revoke access from individual team user accounts.
 
+### Post delivery storage
+
+Users will be able to store their messaging data off-network.
+
+### Signal parity security
```

Comment:
I think this should be more concrete. What are the key security properties implied by this roadmap item?

My concern is that by stating parity is the goal, without specifying what parity actually entails, we are basically just stating that XMTP has less security than Signal.

Reply:
Good point. @richardhuaaa could 'Signal parity' be replaced by explicitly stating the security properties? What would they be?

Reply:
Another approach might be "Implement Signal protocol"

Doesn't require calling out specific security properties or explicitly compare us unfavorably.